packages/opentelemetry-plugin-grpc/test/utils/SpanAuditProcessor.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Span } from '@opentelemetry/types';
+import { SpanAudit } from './SpanAudit';
+
+export class SpanAuditProcessor {
+  private static skipFields = ['_tracer'];
+  private _spans: Span[];
+  constructor() {
+    this._spans = [];
+  }
+
+  push(span: Span) {
+    this._spans.push(span);
+  }
+
+  processSpans(): SpanAudit[] {
+    return this._spans.map(span => {
```

Comment:
Similar to http plugin, I think we should use `ReadableSpan` interface instead of `SpanAdudit`. See: https://github.com/open-telemetry/opentelemetry-js/pull/161/commits/9e6215e0ac5e4ba0bdf52acb6e88cbe301af403b#diff-c5ffc97b1a2d1e3eb08a578bdade4c68R29 WDYT?

Reply:
Updated to use `SpanAudit`